Tests/L0/Gradle/_suite.ts

```diff
@@ -612,12 +638,96 @@ describe('gradle Suite', function() {
             });
     })
 
-    it('Gradle with SonarQube - Should run Gradle with all default inputs when SonarQube analysis disabled', function(done) {
+    it('Gradle with SQ in a PR build - SQ issues mode analysis', function (done) {
+
         // Arrange
-        setResponseFile('gradleGood.json');
+        setResponseAndBuildVars(
+            'gradleSonarQube.json',
```

Comment:
We seem to always use gradleSonarQube.json - is there a risk of dirty reads because of previous tests?

(For the dynamic stuff in the Maven tests I pre-pended this.test.title)